tests/testthat/test-gt_plt_percentile_dot.R

```diff
@@ -51,6 +45,7 @@ test_that("add_pcttile_plot creates a plot", {
   })
 
 test_that("gt_plt_percentile works as intended", {
+  check_suggests()
```

Comment:
I removed rvest (and xml2 and webshot2) and ran the tests. If you're going to rig things to work in the absence of suggested packages, then it seems like a few more skips are needed.